app/views/referee_interface/reference/confirmation.html.erb

```diff
@@ -0,0 +1,11 @@
+<main class="govuk-main-wrapper govuk-main-wrapper--auto-spacing" id="main-content" role="main">
+
+  <%= form_with model: @reference, url: referee_interface_reference_finish_path, method: :get do |f| %>
```

Comment:
This doesn't need to be a form I think?

Reply:
That's because the radio button for the consent to be contacted is coming soon,
so it'll need to be a form.